android/build.gradle

```diff
@@ -59,6 +59,11 @@ buildscript {
             .forUseAtConfigurationTime()
             .isPresent()
 
+    def matdbg = providers
```

Comment:
You should also document that property in the big comment at the top of the file

Reply:
done